### PR TITLE
Bring navbar menu to top left corner

### LINKED ISSUE #139: Fix mobile navbar menu to top left
**Is your feature request related to a problem? Please describe.**
Mobile navbar menu in CircuitVerse's documentation is fixed at bottom left now, making it hard to find. 

**Describe the solution you'd like**
Fix mobile navigation menu bar to top left

**Additional context**
image

### PR DESCRIPTION
**Pull request by SamriddhiMishra**
Fixes #139

### CODE REVIEW

**Comment by SamriddhiMishra** [comment]
I moved the CSS to theme.css and as the link tag which includes vue.css is overriding the CSS for class(sidebar-toggle), hence I simply placed it above the link tag that includes theme.css to make CSS effective.

File: docs/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 <head>
 	
-
+<link id="stylesheet" rel="stylesheet" href="//unpkg.com/docsify/themes/vue.css">
```
Comment (JoshHeng) [inline docs/index.html]: Move this to line 19, between the CSS comment and the other stylesheet. Also keep the identation like the others

File: docs/styles/theme.css [hunk 2]
```diff
@@ -63,3 +62,12 @@ input:checked + .slider:before {
 .slider.round:before {
   border-radius: 50%;
 }
+
+.sidebar-toggle {
+  bottom: initial;
+}
+
+.search
+{
```
Comment (JoshHeng) [inline docs/styles/theme.css]: Check position of bracket - should be `.search {`

File: docs/styles/theme.css [hunk 3]
```diff
@@ -63,3 +62,12 @@ input:checked + .slider:before {
...
+margin-top:39px;
```
Comment (JoshHeng) [inline docs/styles/theme.css]: Check identation

File: docs/styles/theme.css [hunk 4]
```diff
@@ -63,3 +62,11 @@ input:checked + .slider:before {
 .slider.round:before {
   border-radius: 50%;
 }
+
+.sidebar-toggle {
+  bottom: initial;
+}
+
+.search {
+  margin-top:39px;
```
Comment (tachyons) [inline docs/styles/theme.css]: space after `margin-top`

**Pull request merged**